wicketd/src/artifacts/update_plan.rs

```diff
@@ -310,9 +330,12 @@ impl<'a> UpdatePlanBuilder<'a> {
         };
 
         let (rot_a_data, rot_b_data) = Self::extract_nested_artifact_pair(
```

Comment:
`extract_nested_artifact_pair` writes to temp files; should we async-ify it too?

Reply:
Nevermind - I see you made async-friendly via `block_in_place`